.env.test

```diff
@@ -0,0 +1,3 @@
+NODE_ENV=test
+MSSQL_CREDENTIALS:Server=mssql,1433;Database=master;User Id=sa;Password=Pass@word;trustServerCertificate=true;
```

Comment:
This names the database as "master", but below it seems like we call it "test"? I'm wondering where "master" is referenced and if we can change that to something else.